### PR TITLE
Shopify (Independent Connector)

### PR DESCRIPTION
**Pull request by raybennett-msft**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image
image
image

### CODE REVIEW

File: independent-publisher-connectors/Shopify/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1808 @@
...
+                                    "items": {
+                                        "$ref": "#/definitions/order"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "default",
+                        "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Shopify/apiDefinition.swagger.json:175]: @raybennett-msft schema is not allowed within the default response. please remove it. please fix all default responses in the swagger file. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@raybennett-msft your connector files are missing the readme file. please add this file. please see our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).

File: independent-publisher-connectors/Shopify/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Title
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Shopify/readme.md:1]: @raybennett-msft please replace the Title with the connector name please.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @raybennett-msft ,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

File: independent-publisher-connectors/Shopify/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1804 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0.0",
+        "title": "Shopify",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Shopify/apiDefinition.swagger.json:5]: Hello @raybennett-msft,
I hope you are doing well. It turns out that we have a submission for Shopify and it is under development. As you know we can't have the same name for both connectors. May I ask you please to change the name? maybe to "Shopify IP"

File: independent-publisher-connectors/Shopify/readme.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Shopify
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/Shopify/readme.md:1]: @raybennett-msft May I ask you please to change the name? maybe to "Shopify IP"

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@raybennett-msft please change the folder name to "ShopifyIP" as well. This is very important. Thank you so much.
Please accept my apologies for any inconvenience this may cause.

File: independent-publisher-connectors/Shopify/apiDefinition.swagger.json (hunk 3 above)
Comment (raybennett-msft) [reply independent-publisher-connectors/Shopify/apiDefinition.swagger.json:5]: Hi @Amjed-Ayoub I do not see any open PR for Shopify, can you please point me to it? More out of curiosity if we can somehow combine our effort.

File: independent-publisher-connectors/Shopify/readme.md (hunk 4 above)
Comment (Amjed-Ayoub) [reply independent-publisher-connectors/Shopify/readme.md:1]: It is an internal connector, which our engineers are working on and development the connector with Shopify themselves. this connector is in our pipeline for more than 2 years now and it is not open sourced. 
Thank you so much for understanding.

File: independent-publisher-connectors/Shopify/apiDefinition.swagger.json (hunk 3 above)
Comment (Amjed-Ayoub) [reply independent-publisher-connectors/Shopify/apiDefinition.swagger.json:5]: @raybennett-msft It is an internal connector, which our engineers are working on and development the connector with Shopify themselves. this connector is in our pipeline for more than 2 years now and it is not open sourced.
Thank you so much for understanding.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @raybennett-msft ,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by raybennett-msft** [comment]
[like]  Ray Bennett reacted to your message:
________________________________

**Review by Amjed-Ayoub — APPROVED** [review]
Hello @raybennett-msft ,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Pull request merged**